Firestore/core/src/firebase/firestore/util/log_apple.mm

```diff
@@ -50,11 +50,9 @@ FIRLoggerLevel ToFIRLoggerLevel(LogLevel level) {
 void LogMessageV(LogLevel level, NSString* format, ...) {
   va_list list;
   va_start(list, format);
-
+  NSString *formattedMessage = [[NSString alloc] initWithFormat:format arguments:list];
   FIRLogBasic(ToFIRLoggerLevel(level), kFIRLoggerFirestore, @"I-FST000001",
-              format, list);
-
-  va_end(list);
```

Comment:
I think this `va_end` should stay.

Reply:
Oh yes, I meant to move it up like I did to the others. I re-added it in the latest commit.